README.md

```diff
@@ -36,3 +36,13 @@ AWS_SECRET_ACCESS_KEY=<YOUR_AWS_SECRET_ACCESS_KEY>
 Once you have set up your `.env` file, you need to run `./scripts/setup.sh` from the root folder in
 order to setup your credentials. This script will write the appropriate credentials to a
 `~/.aws/credentials` file that will allow you to deploy your application to the ecosystem.
+
+
+## Installing dependences
+
+Make sure you have `Node`, `npm` and `yarn` installed on your local machine otherwise go to
+`https://changelog.com/posts/install-node-js-with-homebrew-on-os-x` and install them; then run `yarn` from the project directory to install all dependences.
+
+## Starting the server
+
+Navigate to the cloned or forked project directory in command line and type `npm start` and the project will automatically run on `localhost:3000`.
```

Comment:
Minor change: "in command line" -> "using the command line, type... and the project..."